### PR TITLE
Week 3 - Functions

### PR DESCRIPTION
**Pull request by smoore87**

### CODE REVIEW

File: Part 2/Week 3/week_3.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    function areaOfCircle(radius){
+        return (radius * radius * PI);
+    }
+
+    function areaOfRectangle(length, width) {
+            var area = length * width;
+            return area;
+    }
+    
+    var userLegnth = prompt("Enter the length of your rectangle");
```
Comment (jd-gray) [inline Part 2/Week 3/week_3.md]: userLength :)

File: Part 2/Week 3/week_3.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    var userRadius = prompt("Enter the radius of your circle");
+    var circleArea = areaOfCircle(userRadius);
+    alert("Your circle area is: " + circleArea);
+</code>
+</pre>
+
+Note that the functions are not executed in the order they are declared! We defined the areaOfCircle() first, but we called areaOfRectangle() first in the execution. If we 
+never called areaOfCircle() at all, the code within its function would never execute. 
+<br/>
+<h4>Exercises:</h4>
```
Comment (jd-gray) [inline Part 2/Week 3/week_3.md]: This is great, we should try to be heavy exercise this class with the review of week 2 and this week

Comment (jd-gray) [reply Part 2/Week 3/week_3.md]: Also when I teach I will probably include a coin flip game with these exercises. Or maybe I will include that in the week 2 review/practice

File: Part 2/Week 3/week_3.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+<ul>
+    <li>Create a function to convert Celcius to Fahrenheit temperature and vice versa. Formulas:</li>
+        <ul>
+            <li>From F to C: (F - 32) * 5/9</li>
+            <li>From C to F: (C * 9/5) + 32</li>
+        </ul>
+    <li>Write a function that counts the number of vowels in a string that the user enters.</li>
```
Comment (dhart1120) [inline Part 2/Week 3/week_3.md]: This one seems difficult if you haven't covered loops yet.

**Pull request merged**